test/test-cases/functional/saic/sai-api/test_sai_api_vnet_pa_entry.py

```diff
@@ -0,0 +1,144 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+
+# Constants
+SWITCH_ID = 5
+
+
+class TestSaiVnetVni:
```

Comment:
The same class name as in the next test module.

Reply:
fixed